src/app/Main/Body/QueryPage/QueryPage-Container.component.tsx

```diff
@@ -75,8 +58,8 @@ export function QueryPageContainerComponent(): JSX.Element {
     const [dataIsMounted, setDataIsMounted] = useState<boolean>(false);
     const [dataIsLoading, setDataIsLoading] = useState<boolean>(true);
     const [
-        conditionalValuesAreLoading,
-        setConditionalValuesAreLoading,
+        updateIsLoading,
```

Comment:
Please use more descriptive names. An update can mean anything in your app.

Reply:
Done.